cloudigrade/util/redhatcloud/sources.py

```diff
@@ -296,3 +297,19 @@ def notify_application_availability(
         message = f"KafkaException: {exception.args[0].str()}"
         logger.exception(exception)
         raise KafkaProducerException(message)
+
+
+def _check_response(error, message):
+    """Verify that the message was delivered without an error."""
+    if error is not None:
+        logger.error(
+            _("Failed to deliver message: %(message)s due to: %(error)s"),
+            {"message": message, "error": error},
+        )
+    else:
+        log_msg = _("Successfully delivered message: %(message)s"), {"message": message}
+
+        if settings.VERBOSE_SOURCES_NOTIFICATION_LOGGING:
+            logger.info(log_msg)
```

Comment:
Does this actually work as expected? I would naively expect we'd need to explode the `log_msg` tuple like 

```suggestion
            logger.info(*log_msg)
```

I tried something similar in a local shell:

```
>>> log_msg = "hello %(foo)s", {"foo": "world"}
>>> logger.info(log_msg)
2021-06-09 18:35:37,496 | INFO | 38488 | <console>:<module>:1 | requestid:None | ('hello %(foo)s', {'foo': 'world'})
>>> logger.info(*log_msg)
2021-06-09 18:35:41,029 | INFO | 38488 | <console>:<module>:1 | requestid:None | hello world